webapp/src/game/game.service.ts

```diff
@@ -0,0 +1,57 @@
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { Game } from "./game.entity";
+import { initGameState, initRandomGame, Board, GameState } from "@kamon/core";
+
+export class GameService {
+  games: Repository<Game>;
```

Comment:
I prefer gameRepository to know quickly what is by just reading the class member.
Games can also designate a list of games.